tox.ini

```diff
@@ -29,6 +29,18 @@ extras =
 commands =
     pytest {posargs}
 
+[testenv:coverage]
+description = Invoke nile-coverage to generate coverage report
```

Comment:
should we add a rule to require PRs to maintain the coverage?

Reply:
I assumed this would be part of the CI action that I have no permission to add (integrating with codecov). I left that part to @frangio (because the solidity contracts repo is also integrated with codecov and this check), but I can work on it if needed.